### ISSUE TITLE
OTU tables must be the json version of biom

### ISSUE THREAD

**Issue by Chriska1** [knights-lab/BugBase#8]
Hi, I am using the web application of BugBase and keep on running into the same error: "There was an error while parsing your data. Please ensure your OTU and mapping files follow the format explained in the documentation. OTU tables must be the json version of biom." We have been troubleshooting this at the Knight lab, but couldn't get to a solution. To our best knowledge, both biom and metadata files are in the right format. It is a merged biom file from 3 separate biom files though, all of which were closed reference against Greengenes. I tried the commands on other datasets, which works, but not this particular biom file. Does someone have time to help diagnose? Thanks, Chris

**Comment by jingao-yu** [comment]
Hi, I have tried to convert BIOM files according to the following website, and that error information disappears:
http://biom-format.org/documentation/biom_conversion.html
commands:
biom convert -i table.biom -o table.from_biom.txt --to-tsv
biom convert -i table.txt -o table.from_txt_json.biom --table-type="OTU table" --to-json

**Comment by Chriska1** [comment]
Thank you Jingao. I have tried this a dozen times; it doesn't work for this particular file, and I cannot figure out why. Can I send you the file so you can try? Thanks, Chris

**Comment by jingao-yu** [comment]
Yes, my pleasure. You can sent it to jingao_yu@sina.cn, let me try.

**Comment by Candice-Q** [comment]
Hi Jingao, were you able to fix the problem? I am having similar issues. I have also tried converting my biom table to a text and back but I receive the same error.

**Comment by hfxfroi** [comment]
Try my solution: first, change the OTU_1, OTU_2, OTU_3 of the first column in OTU_table into numbers (1, 2, 3... do not contain any other characters), and second, convert OTU_table into biom file.

**Comment by snowflake01986** [comment]
Hi, guys, has anyone tried these out...,　Ｉ have the same problem about file format of biom json.

**Comment by hfxfroi** [comment]
I did run it successfully before, but now I don't know what's going on. I can't even run an example of the official website.
  

 

 ------------------ 原始邮件 ------------------
  发件人: "Karl Leswing"<notifications@github.com>;
 发送时间: 2019年8月16日(星期五) 凌晨0:21
 收件人: "knights-lab/BugBase"<BugBase@noreply.github.com>;
 抄送: "hfxfroi"<hfxfroi@qq.com>;"Comment"<comment@noreply.github.com>;
 主题: Re: [knights-lab/BugBase] OTU tables must be the json version of biom(#8)

 

 
Hi, guys, has anyone tried these out...,　Ｉ have the same problem about file format of biom json.

**Comment by snowflake01986** [comment]
It seems that the sample data contain dos newline instead of windows. Fine,  any way, no idea. there seems to be no maintenance staff.

**Comment by snowflake01986** [comment]
sorry, instead of linux
